modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase

```diff
@@ -75,4 +75,37 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11516. If a catalog with a supplier account is created, but the supplier account is deleted, the previously created supplier account will not be linked to the catalog via API"
```

Comment:
When a supplier account is deleted after being linked to a catalog, the account will not be shown as linked to the catalog.